lib/msf/core/exploit/smb/client/pipe_auditor.rb

```diff
@@ -40,6 +40,9 @@ def check_named_pipes(check_first: [], return_first: false)
     named_pipes.each do |pipe|
       begin
         pipe_name   = pipe.strip
+        if !pipe_name.match(/^\\.*$/)
```

Comment:
```suggestion
        unless pipe_name.start_with?("\\")
```

Reply:
Nice sounds good. New to Ruby; very similar to Swift